Sources/BraintreePayPal/BTPayPalRequest.swift

```diff
@@ -96,6 +96,10 @@ import BraintreeCore
     /// :nodoc: Exposed publicly for use by PayPal Native Checkout module. This property is not covered by semantic versioning.
     @_documentation(visibility: private)
     public var paymentType: BTPayPalPaymentType
+    
+    /// Optional: A user's phone number to initiate a quicker authentication flow in the scenario where the user has a PayPal account
+    /// identified with the same phone number.
+    public var userPhoneNumber: String?
```

Comment:
Does it matter if a merchant include `-` characters in this string?

Reply:
I asked this same question earlier today on the android PR, sounds like Tim is tracking down an answer: https://github.com/braintree/braintree_android/pull/1003#discussion_r1601732253

Reply:
great question! yup @tdchow reached out to the gateway api team and let us know that they don't have a specific format (spaces, dashes, hyphens included within the phone number) that they recommend - seems like they have a regex which takes care of handling and removing most of these additional characters.